### PR TITLE
Fix error in docs

### PR DESCRIPTION
**Pull request by ayush1999**

### CODE REVIEW

**Comment by ayush1999** [comment]
I signed it!

**Comment by ayush1999** [comment]
@rsepassi Since this is my first PR to the project, would love a review.

File: docs/datasets.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ tfds.list_builders()
 
 # Load a given dataset by name, along with the DatasetInfo
 data, info = tfds.load("mnist", with_info=True)
-train_data, test_data = data['test'], data['train']
+train_data, test_data = data['train'], data['test']
```
Comment (cyfra) [inline docs/datasets.md]: This file is generated automatically using: "scripts/document_datasets.py".

Please modify the code there.

Comment (ayush1999) [reply docs/datasets.md]: Sure, I wasn't aware that these are generated docs. Updated. :+1:

**Comment by cyfra** [comment]
Thanks for the pull request. We've fixed this issue together with couple others in #130

**Pull request closed without merging**